style/js/shield_text.js

```diff
@@ -102,17 +188,10 @@ export function drawShieldText(ctx, text, textLayout) {
  * @param {*} bannerIndex - plate position to draw, 0=top, incrementing
  */
 export function drawBannerText(ctx, text, bannerIndex) {
-  var textLayout = layoutShieldText(
-    text,
-    {
-      left: 0,
-      right: 0,
-      top: 0,
-      bottom: 0,
-    },
-    { width: ctx.canvas.width, height: ShieldDef.bannerSizeH },
-    rectTextConstraint
-  );
+  var textLayout = layoutShieldTextFromDef(text, null, {
```

Comment:
I see why it's done this way rather than calling layoutShieldText, but it's a bit counterintuititve to call a function named layoutShieldTextFrom**Def**, and then pass the **def** parameter as `null`.  Perhaps the the function names can be made clearer in the future.